qcodes/data/data_set.py

```diff
@@ -11,6 +11,8 @@
 from .location import FormatLocation
 from qcodes.utils.helpers import DelegateAttributes, full_class, deep_update
 
```

Comment:
There is a bunch of old style non named loggers in this file. I think we should convert those too